docs/reference-guides/rancher-webhook.md

```diff
@@ -38,3 +58,34 @@ When using a private GKE cluster, errors may occur that prevent the Kubernetes A
 Internal error occurred: failed calling webhook "rancher.cattle.io.namespaces.create-non-kubesystem": failed to call webhook: Post "https://rancher-webhook.cattle-system.svc:443/v1/webhook/validation/namespaces?timeout=10s": context deadline exceeded
 ```
 This issue occurs because firewall rules restrict communication between the API server and the private cluster. To resolve this communication problem, users must add firewall rules to allow the GKE control plane to communicate with the Rancher-Webhook on port 9443. Please refer to the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules) for detailed information and steps on updating the firewall rules.
+
+### Application Fails to Deploy due to rancher-webhook blocking access
+
+The webhook provides extra validations on [namespaces](https://github.com/rancher/webhook/blob/release/v0.4/docs.md#psa-label-validation). One of these validations ensures that users can only update PSA relevant labels if they have the proper permissions (`updatepsa` for `projects` in `management.cattle.io`). This can result in specific operators (such as tigera or trident) which attempt to deploy namespaces with PSA labels failing to deploy these resources. There are several ways that this issue can be resolved:
```

Comment:
```suggestion
The webhook provides extra validations on [namespaces](https://github.com/rancher/webhook/blob/release/v0.4/docs.md#psa-label-validation). One of these validations ensures that users can only update PSA-relevant labels if they have the proper permissions (`updatepsa` for `projects` in `management.cattle.io`). This can result in specific operators, such as tigera or trident, attempting to deploy namespaces with PSA labels that fail to deploy these resources. There are several ways to resolve this issue:
```

Reply:
Multiple instances of:
`trident` -> `Trident`
`tigera` -> `Tigera`

Reply:
I've made some slight wording changes to the proposed changes - I will leave this un-resolved until we agree